packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/en.ftl

```diff
@@ -1,6 +1,9 @@
 # Variables:
 #  $serviceName (String) - A service the user hasn't signed into before (e.g. Firefox)
-verifyLoginCode-subject-line = Sign-in code for { $serviceName }
-verifyLoginCode-title = Is this you signing in?
-verifyLoginCode-prompt-2 = If yes, here is the confirmation code:
+verifyLoginCode-subject-line-2 = Approve sign in to { $serviceName }
```

Comment:
```suggestion
verifyLoginCode-subject-line-2 = Approve sign-in to { $serviceName }
```
Since sign-in is being used as a noun this should be hyphenated.